### PR TITLE
Fixed references to RC2, OS X, and Update 2

### PR DESCRIPTION
**Pull request by stevehoag**

### CODE REVIEW

**Comment by mairaw** [comment]
Is this the one we should review @stevehoag? I'm confused.

**Comment by stevehoag** [comment]
yes, but I have merge conflicts that I can't seem to fix

**Comment by mairaw** [comment]
@BillWagner can probably help you with that

**Comment by BillWagner** [comment]
@stevehoag Give me 2 minutes, and I'll help with the merge changes. I looked at your PR and fixing the conflicts will take us < 5 minutes over skype

**Comment by mairaw** [comment]
do I need to comment here again or will you look at my comments from PR #614?

**Comment by stevehoag** [comment]
@mairaw I'll reference your comments in #614.

@BillWagner yes, I'd appreciate the help.

**Comment by mairaw** [comment]
@stevehoag awesome! :+1:

File: docs/core/rid-catalog.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ wish to check if new ones are added, please check back here.
     * `linuxmint.17.2-x64`
     * `linuxmint.17.3-x64`
 
-## OS X RIDs
+## macOS RIDs
```
Comment (richlander) [inline docs/core/rid-catalog.md]: This one is probably wrong. These are indeed OS X RIDs.

The rule is:

- Use macOS for the general usage of the operating system.
- Use OS X if you are talking about El Capitan or earlier.

We don't support macOS Sierra yet, so this is El Capitan.

File: docs/core/tools/dotnet-publish.md [hunk 2]
```diff
@@ -88,5 +88,5 @@ Publishes the current application using the `netcoreapp1.0` framework.
 	
 `dotnet publish --framework netcoreapp1.0 --runtime osx.10.11-x64`
     
-Publishes the current application using the `netcoreapp1.0` framework and runtime for `OS X 10.10`. This RID has to 
+Publishes the current application using the `netcoreapp1.0` framework and runtime for `macOS 10.10`. This RID has to
```
Comment (richlander) [inline docs/core/tools/dotnet-publish.md]: I suspect that this one should stay as OS X, too.

I strongly suspect that these commands will fail if you tried them with "macOS" and they should unless you are using macOS Sierra.

File: docs/core/rid-catalog.md (hunk 1 above)
Comment (stevehoag) [inline docs/core/rid-catalog.md]: fixed

File: docs/core/tools/dotnet-publish.md (hunk 2 above)
Comment (stevehoag) [inline docs/core/tools/dotnet-publish.md]: fixed

**Comment by stevehoag** [comment]
Can I merge this now (before more files get moved)?

**Comment by BillWagner** [comment]
:shipit:

**Pull request merged**